init.rb

```diff
@@ -98,6 +100,9 @@ def display_dyno_type_and_costs(app_resp, formation_resp)
       }
     end
 
+    # in case of an app not yet released
```

Comment:
the exact term a few lines above was 'when the app is on production and performance' - seemed misleading in our context, as this is a sandbox of ours, running in staging env. I guess it needs some code pushed, with a Procfile, and then the app scaled. All of these are missing for us at this time, as we are doing it before setting up the sandbox. I guess who owns the repo should know about their preferred terms, i'm happy to correct it.